### DISCUSSION TITLE
Cannot upload proof image after my Previous Request was declined for an unrelated issue

### DISCUSSION THREAD

**Discussion by mouseylicense** [community/community#139102] [GitHub Education]
### Hi everyone,

Hello, Im Trying to apply for a The Student Developer Pack, My previous request was denied because i didnt specify My School's Name (which there was no field for, btw) , now i went to reapply and i cant upload my image of proof, because i already uploaded it Once. 
What can i do?

**Comment by alx-mp4** [comment]
Hi @mouseylicense, I recommend opening a ticket with GitHub support for this issue. I had applied the same issue and the GitHub support ticket gave me insight and resolved my issue.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
